.package-version

```diff
@@ -1 +1 @@
-8.16.0
\ No newline at end of file
+9.0.0
```

Comment:
This normally is handled by the automation ---> https://github.com/elastic/elastic-agent/pull/5588/
since it would be a chicken and egg problem I guess we can update `.package.version` here and close the other PR once we merge this